sonic-net/sonic-buildimage
# Events: Degrade log level to info for network related.

#### Why I did it
LogAnalyzer in test runs take ERR log messages as failure.
Eventd reports some network related issues as error, which could happen due to external network state.

#### How I did it
Reduce ERR to INFO.

#### How to verify it
Per issue, "config reload" triggers it,
Do config-reload and ensure no ERR message from eventd.
[Issue #12303](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsonic-net%2Fsonic-buildimage%2Fissues%2F12303&data=05%7C01%7CRenuka.Manavalan%40microsoft.com%7C623b4b31db7e42330f5208daaa8d0eba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638009820084595812%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=dec6xkVNi8RTZFaoqHqLYijw9qQCigaC9fBGMIcAea4%3D&reserved=0) 
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
